cdp_scrapers/types.py

```diff
@@ -14,7 +16,9 @@ class ContentURIs(NamedTuple):
     caption_uri: Optional[str] = None
 
 
-class ScraperStaticData(NamedTuple):
+@dataclass_json
+@dataclass
+class ScraperStaticData:
```

Comment:
This is a new styling thing: you should use the `DataClassJsonMixin` class inheritance rather than the class decorator. We get typing complaints if we use the decorator.

```python
from dataclasses_json import DataClassJsonMixin

class ScraperStaticData(DataClassJsonMixin):
```

Reply:
Made the change.